### PR TITLE
Performance improvements for GAME scoring

### PR DESCRIPTION
**Pull request by joshvfleming**

### CODE REVIEW

File: photon-client/src/main/scala/com/linkedin/photon/ml/cli/game/scoring/Params.scala [hunk 1]
```diff
@@ -260,6 +267,12 @@ object Params {
             }
         )
 
+      opt[Boolean]("spill-scores")
+        .text(s"Whether to spill persisted scores to disk. This is more CPU intensive, but prevents recomputation " +
+          "of scores when memory-persisted blocks are evicted. Useful far very large scoring tasks. " +
```
Comment (fastier-li) [inline photon-client/src/main/scala/com/linkedin/photon/ml/cli/game/scoring/Params.scala]: "for" instead of "far"?

**Comment by fastier-li** [comment]
Can you try on a couple of larger datasets on the grid? I'm not sure the change would be adequately tested by the unit or even integration tests we have.

File: photon-client/src/main/scala/com/linkedin/photon/ml/cli/game/scoring/Params.scala (hunk 1 above)
Comment (joshvfleming) [reply photon-client/src/main/scala/com/linkedin/photon/ml/cli/game/scoring/Params.scala]: Yep.

**Comment by joshvfleming** [comment]
@fastier-li I'm going to leave this up to the folks requesting the change, since I don't really have a case that could adequately reproduce / test the problem.

**Pull request merged**